data/json/items/armor/shields.json

```
@@ -84,5 +84,79 @@
     "environmental_protection": 2,
     "techniques": [ "WBLOCK_2" ],
     "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN" ]
+  },
+  {
+    "id": "shield_leather",
+    "type": "ARMOR",
+    "name": { "str": "leather reinforced shield" },
+    "description": "A simple round shield made of wood with leather reinforcement.  Sturdier and stronger than a simple wooden shield, and less encumbering than a metal one.",
+    "weight": "3200 g",
+    "volume": "3 L",
+    "price": "80 USD",
+    "price_postapoc": "8 USD",
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "wood" ],
```

Figured as with Medieval Mod Reborn's version it'd be useful to have an explanation in a comment.
```suggestion
    "//": "Wood-only to prevent repairing it via sewing.",
    "material": [ "wood" ],
```